Terms of Reference.md

```diff
@@ -0,0 +1,140 @@
+# Terms of Reference
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Project Scope](#project-scope)
+- [Purpose of the Technical Committee](#purpose-of-the-technical-committee)
+- [Membership Composition](#membership-composition)
+- [Project Schedule](#project-schedule)
+- [Meeting Schedule](#meeting-schedule)
+- [Consensus Building](#consensus-building)
+- [Code of Conduct](#code-of-conduct)
+
+## Overview
+
+This Terms of Reference (ToR) defines the goals of the Trust Meter Technical
+Specification, how it differs or relates to other specifications regarding
+Artificial Intelligence (AI) and Trust, and the objectives of the Technical
+Committee, outlining the roles and responsibilities of those involved.
+
+## Project Scope
+
+The Trust Meter Technical Specification addresses statistical discrimination
+against data outliers and small minorities in mechanized statistical reasoning
+in AI decision tools. The Trust Meter Technical Specification is intended to
+alert AI operators when the scenario, group or individual about whom the
+decision is made is out-of-distribution relative to the training set the model
+is trained on, meaning the AI should not be trusted to make decisions in this
+context.
+
+## Purpose of the Committee
+
+The overall objective of the Technical Committee is to determine how to evaluate
+an AI tool's decision making algorithm when used with marginalized or outlier
+individuals; to detect whether the use of the AI discriminates against them.
+This main objective leads to some sub-goals.
+
+The committee will develop definitions of "marginalized" and "outlier" to fully
+contextualize their importance and relevance to the evaluation of an AI model.
```

Comment:
According to my previous comment, would it make sense to replace "AI model" with "machine learning model"?

Reply:
I added a new paragraph at the top of the "Purpose of the Technical Committee" section describing the type of AI under consideration.